chapters/packages.tex

```diff
@@ -171,6 +171,8 @@ \section{The Modelica Library Path -- MODELICAPATH}\label{the-modelica-library-p
 This subdirectory must have a file \filename{package.mo} containing a definition of the package \lstinline!MyLib!, according to the Modelica rules on how to map a package hierarchy to the file system.
 The subpackage \lstinline!Pack2! is stored in its own subdirectory or file in the subdirectory \filename{MyLib}.
 In this case the search succeeds and the package \lstinline!MyLib.Pack2! is loaded into the environment.
+
+Note that if you use the environment variable \lstinline!MODELICAPATH! in Modelica the value is a string which implies that it is quoted, i.e., \lstinline!"C:\\library;C:\\lib1;C:\\lib2"!.
```

Comment:
I don't think this is the correct way to phrase it; isn't the quoting just a syntax of defining a string environment variable in some shell?  I propose to just skip it:
```suggestion
```

Reply:
It is about an environment variable.

They are not restricted to shells and you can set (and get) environment variables in Modelica using `Modelica.Utilities.System.setEnvironmentVariable` and `Modelica.Utilities.System.getEnvironmentVariable("MODELICAPATH")`;

Reply:
> you can set (and get) environment variables in Modelica using `Modelica.Utilities.System.setEnvironmentVariable` and `Modelica.Utilities.System.getEnvironmentVariable("MODELICAPATH")`;

That is in the standard library though, not the language. You can imagine writing external functions that represent environment variables as integers if you feel like it.

Can you explain what you mean with the proposed sentence `use the environment variable MODELICAPATH in Modelica`? I'm assuming the `Modelica` refers to the language.

Reply:
That Modelica string literals are delimited by double quotes does not mean that all strings in Modelica start and end in `"`.  Similarly, the `MODELICAPATH` is generally not a string starting and ending in `"`, and explaining why double quotes are needed to construct string literals in some contexts is out of toping when talking about `MODELICAPATH`.

Reply:
> That Modelica string literals are delimited by double quotes does not mean that all strings in Modelica start and end in `"`.

??? The syntax for constructing a string literal in the Modelica language start with a double quote. So, I cannot see why this is controversial at all - but as it is just an extra explanation that isn't essential it seems simplest to just remove it.